manifest.json

```diff
@@ -22,13 +22,14 @@
     },
 
     "web_accessible_resources": [
-        "images/jira.png"
+        "images/jira.png",
+        "src/templates.js"
```

Comment:
Is this a remnant of something extra you were working on?

Reply:
Gah. It is :)